doc/developer-documentation/forgae-cli/install.md

```diff
@@ -0,0 +1,25 @@
+# Installing and Help
+
+**ForgAE** is an aeternity framework which helps with setting up a project.
+The framework makes the development of smart contracts in the aeternity network pretty easy. It provides commands for compilation, deployment of smart contracts, running a local node, local compiler and unit testing the contracts.
+
+The framework can be installed via npm:
+```
+npm i -g forgae
+```
+
+## Help
+
+```text
+forgae help
+```
+
+Run this command to give you all possible commands of `forgae` + help info
+
+## Version
+
+```text
+gorgae --version
+```
+
```

Comment:
this should be checked if it will work after the separation, probably it will  give you only the CLI version